zcash_primitives/src/transaction/tests.rs

```diff
@@ -23,6 +23,18 @@ use super::{
 #[cfg(feature = "zfuture")]
 use super::components::tze;
 
+pub fn create_test_asset(asset_desc: &str) -> orchard::note::AssetBase {
```

Comment:
this function will be better utilized  if returns the burn tupple:
```rust
get_burn_tupple(asset_desc: &str, amount u64) -> (AssetBase, Amount) {
```

Reply:
please add a comment that the function is deterministic by using a static `sk_iss`.

Reply:
Done.